src/fast_obj/fast_obj_export.h

```diff
@@ -0,0 +1,54 @@
+#include <stdio.h>
+#include "fast_obj.h"
+
+int fast_obj_write(fastObjMesh* mesh, const char* path)
+{
+    FILE *fout;
+
+    if (!(fout = fopen(path, "w")))
+    {
+        printf("Error!");
+        return 0;
```

Comment:
**issue:** 0 for an error status is kinda misleading :D

Reply:
Yeah, you're right. -1 or 2 as in errno?

Reply:
Not sure, I'd say 2, but I have no logical argument going for it.